frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -165,9 +170,39 @@ const CustomNodeTable: React.FC<CustomNodeTableProps> = ({
     setNodes(preSelectedNodes);
   }
 
+  const hasMinimumCPU = (node: formattedNodeType): boolean => {
+    return convertToBaseValue(node.cpuCapacity) < 16;
```

Comment:
You need to reverse the logic with the name change.

```suggestion
    return convertToBaseValue(node.cpuCapacity) >= 16;
```